### DISCUSSION TITLE
Bicycling with OSM on preloaded track stops too often

### DISCUSSION THREAD

**Discussion by Chiel3** [osmandapp/OsmAnd#12613] [Maps Related]
We do a lot of bicycling with OSMAND with downloaded tracks and during that we experienced some problems

When we make a short stop and start again to continue OSM asks to press “Restart” . When you answer “Yes” then then the program restarts in the automobile modus. That leads to errors in the routing.
Then you have to stop again and press the “Bicycle” icon, that’s a nuisance especially when the weather is colder and your fingers are more stiff.

And why should the program   ask to restart  even after a  stop from a few minutes ?

Chiel Geeraert
The Netherlands

**Comment by vshcherb** [comment]
It doesn't look correct app shouldn't "Restart" the navigation and should only continue it. You can pause navigation in Notification bar and then resume. 
By default the app doesn't pause / stop navigation. Please share screenshot of "Restart" to have more information what happened

**Comment by tommycrock** [comment]
> then the program restarts in the automobile modus.

I'm not sure but this part may be to do with your default profile being automobile. When navigation stops it switches to the default profile so if you change that to bike in settings > osmand settings > default profile it may work around this part of the problem. As vscherb says, it isn't usual behaviour for it to pause navigation when you stop moving. Imagine every time a car driver got stuck in traffic!

**Comment by Chiel3** [comment]
Thank you for your answer.  I changed the default settings to “bicycle”. That’s  solved.

I just don’t  know why it comes up with the question “Continue ?”  with every short stop. Maybe only a formal question ? I give it a try with no action if we stop on a trip.

Best regards,

A. Geeraert
Oostburg, the Netherlands


Verzonden vanuit Mail<https://go.microsoft.com/fwlink/?LinkId=550986> voor Windows

Van: ***@***.***>
Verzonden: maandag 30 augustus 2021 21:50
Aan: ***@***.***>
CC: ***@***.***>; ***@***.***>
Onderwerp: Re: [osmandapp/OsmAnd] Bicycling with OSM on preloaded track stops too often (#12613)


then the program restarts in the automobile modus.

I'm not sure but this part may be to do with your default profile being automobile. When navigation stops it switches to the default profile so if you change that to bike in settings > osmand settings > default profile it may work around this part of the problem. As vscherb says, it isn't usual behaviour for it to pause navigation when you stop moving. Imagine every time a car driver got stuck in traffic!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fosmandapp%2FOsmAnd%2Fdiscussions%2F12613%23discussioncomment-1255872&data=04%7C01%7C%7C45b1638b4a1d4a71006508d96bef72a9%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637659498425166992%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=JsIeYESjWTHQjrb3n5s%2Ba1Xoc2I1jV0YyIxulwofJfw%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAVMAYESOSVWLZEODGLJZTHTT7POJDANCNFSM5C55PNVQ&data=04%7C01%7C%7C45b1638b4a1d4a71006508d96bef72a9%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637659498425176937%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=PGwd2S%2Bi39UM4%2FPbuX4bDvqJHTiyepp4WaqDEvChZ0Y%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C45b1638b4a1d4a71006508d96bef72a9%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637659498425176937%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0s39PltI6UrkOT257AzGZBPMvafsQ62tDgPgWaaCvhQ%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C45b1638b4a1d4a71006508d96bef72a9%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637659498425186908%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=tXHg2nkjVDxyynrzkajbyGhZIV1Y20AyYSkYkjFBUqo%3D&reserved=0>.
